modules/library/data/src/test/java/org/geotools/data/memory/MemoryDataStoreBoundsTest.java

```diff
@@ -53,8 +56,9 @@ protected void setUp() throws Exception {
         data = new MemoryDataStore();
         data.addFeatures(roadFeatures);
         
-        SimpleFeatureType riverTypeWithCrs = SimpleFeatureTypeBuilder.retype(riverType, DefaultGeographicCRS.WGS84);
+        SimpleFeatureType riverTypeWithCrs = SimpleFeatureTypeBuilder.retype(riverType, CRS.decode("EPSG:4326"));
```

Comment:
Some of the comparaison tests were failing with the previous configuration (Specifically assertEquals against the features and the CRSs). While they may be the same mathematically, they have slightly different WKT representations. Consequently, I decided to stick with EPSG:4326 for consistency.